Apps/W1/HybridGP/app/src/codeunits/GPCloudMigration.codeunit.al

```diff
@@ -134,4 +136,26 @@ codeunit 4025 "GP Cloud Migration"
         DataMigrationEntity.InsertRecord(Database::Item, HelperFunctions.GetNumberOfItems());
         exit(true);
     end;
+
+    [EventSubscriber(ObjectType::Codeunit, Codeunit::"Hybrid Cloud Management", 'OnInsertDefaultTableMappings', '', false, false)]
+    local procedure OnInsertDefaultTableMappings(DeleteExisting: Boolean; ProductID: Text[250])
```

Comment:
This will result in merge conflict between you and Todd. Pull and resolve before submitting the final iteration.